Video Player Application/README.md

```diff
@@ -0,0 +1,69 @@
+# Introduction
+
+Hello Geeks, We all use Youtube for watching videos for fun, learning etc. In this microbyte, we will going to make the app like youtube but little different. we will just going to make a single activity playing Video.

```

Comment:
we will be going to make an app like YouTube*
The last sentence doesn't make sense to me. Correct it